sms_939/models/sms_child_request.py

```diff
@@ -68,7 +68,8 @@ def _take_child_from_website(self):
             self.write({"child_id": child.id, "state": "child_reserved"})
             child.hold_id.sms_request_id = self.id
             child.remove_from_wordpress()
-            child.write({"state": "S"})
+            if self.type == "sms":
```

Comment:
This would be needed for all request types. You could as well change the state name in the child from "Reserved for SMS" to "Reserved for Mobile".